docs/preview/features/servicebus-messsage-pump.md

```diff
@@ -0,0 +1,179 @@
+﻿---
+title: Testing Azure Service Bus message handling
+layout: default
+---
+
+# Testing Azure Service Bus message handling
+
+## Installation
+The following functionality is available when installing this package:
+
+```shell
+PM> Install-Package -Name Arcus.Testing.Messaging.Pumps.ServiceBus
+```
+
+## Test Azure Service Bus message pump
+As an addition on the [Arcus Azure Service Bus message pump](), we have provided a test version of the message pump to verify your custom Azure Service Bus message handler implementations.
```

Comment:
Wait until https://github.com/arcus-azure/arcus.messaging/pull/337 is finished before we can update this.

Reply:
That PR is merged, so this can be updated.

Reply:
Yes, updated in https://github.com/arcus-azure/arcus.testing/pull/68/commits/9c4f41d773e52ec70a30fcca955bd05ae45b79c8
👍